packages/lib-user/src/components/shared/ContentBox/components/ContentHeading.js

```diff
@@ -0,0 +1,37 @@
+import { SpacedText } from '@zooniverse/react-components'
+import { Heading } from 'grommet'
+import { node } from 'prop-types'
+import styled from 'styled-components'
+
+const StyledHeading = styled(Heading)`
+  font-size: 16px;
+  line-height: normal;
+`
+
+function ContentHeading ({
+  children
+}) {
+  return (
+    <StyledHeading
+      level={2}
+      margin='none'
```

Comment:
```suggestion
      margin='0'
```

`none` works for Grommet, but it's not an actual option for the CSS [margin](https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax) property. I don't think `none` will throw a warning to the console, but in places where we've used `margin='none'` or `pad='none'` such as [ZooniverseTalk](https://github.com/zooniverse/front-end-monorepo/blob/c2ae9c51850ebfbf95ee1f272e09726559424fff/packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/ZooniverseTalk.js#L33) in app-project, there's a warning in the dev tool's styling inspector:
<img width="151" alt="Screenshot 2024-01-25 at 3 27 35 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/97cd18d3-c945-4e73-a5cb-315e03a2ba03">